graal/graal-server-starter/pom.xml

```diff
@@ -0,0 +1,133 @@
+
+    <properties>
+        <image.name>skywalking-oap</image.name>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.skywalking</groupId>
+            <artifactId>server-starter</artifactId>
+            <version>${project.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.skywalking</groupId>
+                    <artifactId>meter-analyzer</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.apache.skywalking</groupId>
+                    <artifactId>log-analyzer</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.apache.skywalking</groupId>
+                    <artifactId>server-core</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.apache.skywalking</groupId>
+                    <artifactId>oal-rt</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.apache.skywalking</groupId>
+                    <artifactId>agent-analyzer</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.apache.skywalking</groupId>
+                    <artifactId>library-util</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.skywalking</groupId>
+            <artifactId>envoy-metrics-receiver-plugin-graal-native</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.skywalking</groupId>
+            <artifactId>library-util-graal-native</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.skywalking</groupId>
+            <artifactId>meter-analyzer-graal-native</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.skywalking</groupId>
+            <artifactId>server-core-graal-native</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.skywalking</groupId>
+            <artifactId>log-analyzer-graal-native</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.skywalking</groupId>
+            <artifactId>oal-rt-graal-native</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.graalvm.nativeimage</groupId>
+            <artifactId>svm</artifactId>
+            <scope>provided</scope>
+            <version>22.3.1</version>
+        </dependency>
```

Comment:
Is this required for now? I believe this is for the next stage?

Meanwhile, please manage the dependency version from the bom too.

Reply:
My fault, I should remove it when I am not using GraalVM substitutions.